esphome/components/lvgl/lvgl_esphome.cpp

```diff
@@ -0,0 +1,118 @@
+#include "esphome/core/defines.h"
+#ifdef USE_LVGL
+#include "esphome/core/log.h"
+#include "esphome/core/helpers.h"
+#include "esphome/core/hal.h"
+#include "lvgl_hal.h"
+#include "lvgl_esphome.h"
+
+static const char *const TAG = "lvgl";
+
+namespace esphome {
+namespace lvgl {
+
+lv_event_code_t lv_custom_event;  // NOLINT
+void LvglComponent::setup() {
+  esph_log_config(TAG, "LVGL Setup starts");
```

Comment:
Please stick to the defined macros so there is consistency throughout the codebase.
```suggestion
  ESP_LOGCONFIG(TAG, "LVGL Setup starts");
```